src/transactions/feed/SwapFeedItem.test.tsx

```diff
@@ -124,8 +125,10 @@ describe('SwapFeedItem', () => {
     expect(getElementText(getByTestId('SwapFeedItem/outgoingAmount'))).toEqual('-17.54 cEUR')
   })
 
-  it('still shows balances when feature gate false, hide balances root state true', async () => {
-    jest.mocked(getFeatureGate).mockReturnValue(false)
+  it('still shows balances when feature gate true, hide balances root state true', async () => {
```

Comment:
nit: might be cleaner to include the feature gate name on the test, as it seems to depend on a different one now than before